1)解决 UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-78: ordinal not in range(128)...

# 在py文件的最开始加上:

import sys

reload(sys)
sys.setdefaultencoding('utf8')


2)['1:a','2:b','3:c'] 转换为 {’1′: ‘a’, ’3′: ‘c’, ’2′: ‘b’}
a = {}
b = ['1:a','2:b','3:c']
map(lambda x:a.setdefault(x.split(':')[0], x.split(':')[1]), b)
print a

3)
